jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -270,6 +269,8 @@ async def job_script_delete(job_script_id: int = Query(..., description="id of t
         )
 
     delete_query = job_scripts_table.delete().where(where_stmt)
+    s3man_jobscripts.delete(job_script_id)
```

Comment:
put it in a try/except block and catch `KeyError`, in case job script was not found (i.e., already deleted)

Reply:
well, this log wouldn't be 100% precise because it doesn't mean the the job is already deleted; the job could never exist in our database. Line 271 won't cause error in case we try to delete a record that doesn't exist, so, I think we just put in a try-except block and return 404 in case we see `KeyError`

Reply:
The is a query and a test a few lines above (GitHub is not showing here) that raises 404 if the id is not in the database, so the log suggested should be enought.